# Product Principles — Lenny's Podcast Distilled

> **Source**: 320 Lenny's Podcast episodes, synthesized
> **Purpose**: Core algorithms for product decisions at BREZ

---

## The 10 Foundational Themes

### 1. Activation Is Everything

**Principle**: Get users to value FAST. The "aha moment" determines retention.

**Examples**:
- Slack's "2,000 messages" threshold
- Gamma's 30-second magic

**BREZ Application**:
```
ACTIVATION CHECKLIST:
├── What is BREZ's "aha moment"? → First drink, feel the effect
├── How fast do new customers reach it? → Same day (if product delivered)
├── What blocks activation? → Shipping time, unclear dosing, wrong expectation
└── How do we accelerate it? → Messaging, onboarding, first-use guidance
```

**AI Rule**: When evaluating product/UX changes, always ask: "Does this accelerate time-to-value?"

---

### 2. Storytelling as a Superpower

**Principle**: Compelling narratives align organizations better than documents.

**BREZ Application**:
- The $200B vision IS the story
- "Feel better without feeling worse" IS the narrative
- Building in public IS storytelling

**AI Rule**: When crafting internal comms or external content, lead with narrative, support with data.

---

### 3. Positioning Is Strategy, Not Marketing

**Principle**: April Dunford's 5-step framework starts with competitive alternatives, not messaging.

**The Framework**:
```
1. Competitive alternatives (what would they do without you?)
2. Unique attributes (what do you have they don't?)
3. Value (what does that uniqueness enable?)
4. Target customers (who cares most about that value?)
5. Market category (what context makes your value obvious?)
```

**BREZ Positioning**:
```
1. Competitive alternatives: Alcohol, THC edibles, CBD drinks, nothing
2. Unique attributes: Fast-acting, short-lasting, functional mushrooms + cannabis, no hangover
3. Value: Social/relaxation experience without consequences
4. Target customers: Health-conscious adults who want to feel good
5. Market category: Feel-good tonics (we created the category)
```

**AI Rule**: When evaluating new products or messaging, run through this framework. Positioning drift = confusion.

---

### 4. Product Discovery Before Delivery

**Principle**: Teresa Torres' Opportunity Solution Trees + Marty Cagan's Four Risks. Talk to customers weekly, not quarterly.

**The Four Risks**:
```
1. Value risk: Will customers buy/use it?
2. Usability risk: Can they figure out how to use it?
3. Feasibility risk: Can we build it?
4. Business viability risk: Does it work for the business?
```

**AI Rule**: Before any product decision, explicitly assess all four risks. Missing one = blind spot.

**BREZ Application**: Customer voice data should inform product decisions weekly, not after launches.

---

### 5. Ruthless Prioritization

**Principle**: Shreyas Doshi's LNO Framework + opportunity cost thinking.

**LNO Framework**:
```
L = LEVERAGE tasks (10x impact, do these yourself, do them great)
N = NEUTRAL tasks (1x impact, do them adequately)
O = OVERHEAD tasks (0.1x impact, do them fast or delegate)
```

**Opportunity Cost Rule**: Don't ask "What's the ROI?" Ask "What am I NOT doing by doing this?"

**BREZ Application**:
```
PRIORITIZATION FILTER:
├── Is this L, N, or O?
├── What's the opportunity cost?
├── Does this move the bottleneck? (Cash → Demand → Clarity → Focus)
└── If not L or bottleneck-related → DEFER
```

**AI Rule**: Flag O-tasks that are consuming L-task energy. Protect Aaron's leverage time.

---

### 6. AI Is Transforming Product Development

**Principle**: Model maximalism, evals as core PM skill, CEOs becoming ICs again.

**Implications for BREZ**:
- The Supermind IS this transformation
- Product decisions increasingly AI-assisted
- Speed of iteration dramatically increased

**AI Rule**: Continuously ask "Can AI do this faster/better?" for any process.

---

### 7. Empowered Teams vs. Feature Factories

**Principle**: Give teams problems to solve, not features to build.

**The Test**:
```
Feature Factory: "Build a loyalty program"
Empowered Team: "Increase repeat purchase rate by 20%"
```

**BREZ Application**: Frame goals as outcomes, let teams find the path.

**AI Rule**: When logging goals/tasks, check: Is this an outcome or an output? Reframe outputs as outcomes.

---

### 8. Build Your Operating System

**Principle**: Claire Hughes Johnson's framework for founding documents, operating cadence, decision frameworks.

**BREZ Already Has This**:
- CLAUDE.md = Founding document
- Weekly ELT / Monthly LT = Operating cadence
- NO-gate + Governance = Decision frameworks

**AI Rule**: Continuously refine the OS. Every session should make it smarter.

---

### 9. Pre-Mortems and Risk Anticipation

**Principle**: Shreyas Doshi's technique to surface failure modes BEFORE they happen.

**Pre-Mortem Protocol**:
```
Before major decision:
1. Assume it failed spectacularly
2. Write the story of WHY it failed
3. Work backwards to prevent those failures
4. Decide if risk is acceptable
```

**AI Rule**: For any major product/strategy decision, run pre-mortem. Surface the failure modes.

---

### 10. Small Teams, Outsized Impact

**Principle**: Jason Fried's 2-person/6-week constraints. Shopify's pair programming.

**The Math**: Coordination overhead grows O(n²). Fewer people = more output.

**BREZ Application**:
- Resist headcount as solution
- Prefer capability over capacity
- AI augments small teams to punch above weight

**AI Rule**: When "we need more people" comes up, first ask: "What would we do with 2 people and 6 weeks?"

---

## The 10 Counterintuitive Truths

### 1. Fear Gives Bad Advice — Do the Opposite

**Truth**: Whatever you're afraid to do (hard conversation, telling the board bad news) is exactly what you should do.

**BREZ Application**: The AP conversation, the hard calls, the uncomfortable truths. Lean in.

**AI Rule**: When detecting avoidance patterns, flag them. "You seem to be avoiding X. Fear usually means you should do X."

---

### 2. Adding Friction Can INCREASE Conversion

**Truth**: Amplitude added personalization questions to signup → 5% conversion increase.

**Why It Works**: Friction that creates investment/personalization increases commitment.

**BREZ Application**: The subscription model IS friction that increases LTV. Quiz funnels might outperform direct purchase.

**AI Rule**: Don't assume all friction is bad. Ask: "Does this friction create value or investment?"

---

### 3. Fewer Features = More Value

**Truth**: Walkman succeeded because Sony REMOVED recording. QuickBooks wins with half features at double price.

**BREZ Application**:
- Started with ONE flavor, ONE dose
- SKU rationalization > SKU expansion
- Simplicity IS the product

**AI Rule**: Default to REMOVE before ADD. The NO-gate should be harder to pass for additions than subtractions.

---

### 4. Adding People Makes You Slower (Absolutely)

**Truth**: Companies produce MORE total output after layoffs. Coordination overhead is the silent killer.

**BREZ Application**: Resist hiring as the solution. AI + lean team > bloated team.

**AI Rule**: When headcount increase is proposed, calculate coordination cost. Flag if net output likely decreases.

---

### 5. What Customers Say They Want Is Meaningless

**Truth**: 93% said they wanted energy-efficient homes. Nobody bought them. "Bitchin' ain't switchin'."

**BREZ Application**: Watch what customers DO (purchase, repeat, refer), not what they SAY.

**AI Rule**: Weight behavioral data (purchases, retention) over stated preferences (surveys, reviews) in product decisions.

---

### 6. Goals Are Not Strategy — They're the Opposite

**Truth**: Richard Rumelt says confusing goals for strategy is the most common strategic error. OKRs are often just wish lists.

**Strategy vs Goals**:
```
GOAL: "Reach $75M revenue"
STRATEGY: "Couple DTC demand to retail velocity via 8-week lag,
           compound subscription revenue, use retail CM to fund growth"
```

**AI Rule**: When goals are set without strategy, flag it. "This is a goal. What's the strategy to achieve it?"

---

### 7. Don't A/B Test Your Big Bets

**Truth**: Instagram and Airbnb actively reject testing for transformational changes. You can't A/B test your way to greatness.

**When to Test vs. When to Bet**:
```
A/B TEST: Button color, headline copy, price point
BIG BET: New product line, brand repositioning, category creation
```

**BREZ Application**: BREZ itself was a big bet. Elevate, Drift, new states = conviction bets, not tests.

**AI Rule**: Distinguish optimization (test) from transformation (bet). Don't apply testing mentality to bets.

---

### 8. Your Gut IS Data

**Truth**: Intuition is compressed experiential learning that isn't statistically significant yet.

**BREZ Application**: Aaron's GTM intuition has proven prophetic. Don't dismiss it for "data."

**AI Rule**: When Aaron's intuition conflicts with data, flag it — but don't override. Intuition may be seeing what data hasn't captured yet.

---

### 9. By the Time You're Thinking About Quitting, It's Too Late

**Truth**: Stewart Butterfield killed Glitch while it was still growing 6-7% weekly. That's why he could start Slack.

**BREZ Application**:
- SKU rationalization before SKUs die
- Channel exit before channels drain
- Pivot signals before crisis

**AI Rule**: Track leading indicators of decline. Flag "still growing but slowing" — the time to decide is BEFORE the crisis.

---

### 10. Most PMs Are Overpaid and Unnecessary

**Truth**: Marty Cagan says feature teams don't need PMs. Nikita Bier calls PM "not real."

**The Distinction**:
```
VALUABLE PM: Owns outcome, has authority, makes decisions
UNNECESSARY PM: Coordinates, documents, facilitates (AI can do this)
```

**BREZ Application**: AI handles coordination. Humans handle judgment, relationships, conviction.

**AI Rule**: Automate PM coordination work. Preserve PM judgment work. Never confuse the two.

---

## Decision Algorithms

### New Product Idea Filter

```
NEW PRODUCT IDEA:
│
├── Does it improve CM? (Master Governing Principle)
│   └── NO → REJECT
│
├── Does it require ADDING or REMOVING?
│   └── ADDING → Apply 2x scrutiny (Fewer Features = More Value)
│
├── Run Four Risks assessment:
│   ├── Value risk: Will customers buy it?
│   ├── Usability risk: Can they use it?
│   ├── Feasibility risk: Can we build it?
│   └── Viability risk: Does it work for BREZ?
│   └── ANY HIGH RISK → Flag, require mitigation plan
│
├── Run Pre-Mortem:
│   └── What's the story of how this fails?
│   └── Are those failures preventable?
│
├── Is this a TEST or a BET?
│   ├── TEST → Define success metrics, run experiment
│   └── BET → Requires conviction, Aaron approval
│
└── APPROVED → Assign owner, metric, deadline
```

### Feature Request Filter

```
FEATURE REQUEST:
│
├── What problem does this solve? (Outcome, not output)
│   └── If no clear problem → REJECT
│
├── Does this accelerate time-to-value? (Activation)
│   └── NO → Deprioritize
│
├── Is this L, N, or O? (LNO Framework)
│   ├── L → Prioritize
│   ├── N → Queue
│   └── O → Automate or skip
│
├── Can we solve this by REMOVING instead?
│   └── YES → Prefer removal
│
└── What's the opportunity cost?
    └── What are we NOT doing by doing this?
```

---

## Weekly Product Review Prompts

Ask these every week:

1. "What's our current time-to-activation? How do we shorten it?"
2. "What feature could we REMOVE to increase value?"
3. "What are customers DOING (not saying)?"
4. "What's the opportunity cost of our current priorities?"
5. "What are we afraid to do? (That's probably what we should do.)"
6. "Is anything showing 'still growing but slowing' signals?"

---

## Integration with BREZ Operating System

These principles connect to existing rules:

| Principle | Connects To |
|-----------|-------------|
| Activation Is Everything | UX Principles (time-to-value) |
| Ruthless Prioritization | Bottleneck Stack, LNO |
| Fewer Features = More Value | NO-gate (harder for additions) |
| Pre-Mortems | Governance (risk assessment) |
| Fear = Do It | Aaron Kryptonite Protection |
| Gut IS Data | AI Calibration (intuition ≠ override) |
| Small Teams | Team Governance (resist headcount) |

---

*This file evolves. Add learnings from product decisions. Update when new principles emerge.*
